code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -1,1053 +1,656 @@
-					if(istype(L.dna.species, /datum/species/slime))
-						on_CD = handle_emote_CD()
-						found_slime_bodypart = TRUE
-						break
-
-			if(!found_slime_bodypart)								//Everyone else fails, skip the emote attempt
-				return
-
-		if("clack", "clacks")
-			if(iskidan(src))	//Only Kidan can clack and rightfully so.
-				on_CD = handle_emote_CD(30)			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-
-		if("click", "clicks")
-			if(iskidan(src))	//Only Kidan can click and rightfully so.
-				on_CD = handle_emote_CD(30)			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-
-		if("creaks", "creak")
-			if(isdiona(src)) //Only Dionas can Creaks.
-				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-
-		if("hiss", "hisses")
-			if(isunathi(src)) //Only Unathi can hiss.
-				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-
-		if("quill", "quills")
-			if(isvox(src)) //Only Vox can rustle their quills.
-				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-
-		if("warble", "warbles")
-			if(isskrell(src)) //Only Skrell can warble.
-				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-		if("rattle", "rattles")
-			if(isskeleton(src) || isplasmaman(src)) //Only Plasmamen and Skeletons can rattle.
-				on_CD = handle_emote_CD()			//proc located in code\modules\mob\emote.dm'
-			else								//Everyone else fails, skip the emote attempt
-				return
-
-		if("flap", "flaps", "aflap", "aflaps","flutter", "flutters")
-			if(!ismoth(src))
-				return
-			on_CD = handle_emote_CD()
-
-		if("scream", "screams")
-			on_CD = handle_emote_CD(50) //longer cooldown
-		if("fart", "farts", "flip", "flips", "snap", "snaps")
-			on_CD = handle_emote_CD()				//proc located in code\modules\mob\emote.dm
-		if("cough", "coughs", "highfive")
-			on_CD = handle_emote_CD()
-		if("gasp", "gasps")
-			on_CD = handle_emote_CD()
-		if("deathgasp", "deathgasps")
-			on_CD = handle_emote_CD(50)
-		if("spin", "spins")
-			on_CD = handle_emote_CD(50)
-		if("sneeze", "sneezes")
-			on_CD = handle_emote_CD()
-		if("clap", "claps")
-			on_CD = handle_emote_CD()
-		if("slap", "slaps")
-			on_CD = handle_emote_CD(3 SECONDS)
-		//Everything else, including typos of the above emotes
+			user.visible_message("<b>[user.name]</b> and <b>[L.name]</b> high-five!")
+			playsound('sound/effects/snap.ogg', 50)
+			user_carbon.remove_status_effect(STATUS_EFFECT_HIGHFIVE)
+			L.remove_status_effect(STATUS_EFFECT_HIGHFIVE)
+			return
```

Comment:
return TRUE. you don't want to be returing `null` it should be TRUE or FALSE to make it more easily readable.